src/main/java/net/sf/log4jdbc/StatementSpy.java

```diff
@@ -34,6 +36,25 @@
  */
 public class StatementSpy implements Statement, Spy
 {
+
+  static final String SQL_SHOW_PARAMS_KEY = "LOG4JDBC_SQL_SHOW_PARAMS";
+
+  static final boolean showParams;
+
+  static {
+    showParams = getConfigProp(SQL_SHOW_PARAMS_KEY, v -> {
+      switch (v.trim().toLowerCase()) {

```

Comment:
Can't you use `Boolean.parseBoolean`?

Reply:
I just do not see a reason to do it. With switch it is very obvious in my opinion because `Boolean.parseBoolean` requires additional checks for invalid inputs.